index.bs

```diff
@@ -3459,7 +3548,109 @@ The {{MLInstanceNormalizationOptions}} members are:
 </details>
 </div>
 
-### The leakyRelu() method ### {#api-mlgraphbuilder-leakyrelu}
+### layerNormalization ### {#api-mlgraphbuilder-layernorm}
+Normalize the input using [[Layer-Normalization]]. Unlike [[#api-mlgraphbuilder-batchnorm]] where the mean and variance values are computed across all the samples in the batch dimension while the model is trained, and in [[#api-mlgraphbuilder-instancenorm]] where the mean and variance values are computed on the fly for each input feature of each individual sample in the batch, the means and variance values of the layer normalization are computed on the fly across all the input features of each individual sample in the batch.
+
+<script type=idl>
+dictionary MLLayerNormalizationOptions {
+  MLOperand scale;
+  MLOperand bias;
+  sequence<unsigned long> axes;
+  float epsilon = 1e-5;
+};
+
+partial interface MLGraphBuilder {
+  MLOperand layerNormalization(MLOperand input, optional MLLayerNormalizationOptions options = {});
+};
+</script>
+
+The {{MLLayerNormalizationOptions}} members are:
+<dl dfn-type=dict-member dfn-for=MLLayerNormalizationOptions>
+    : <dfn>scale</dfn>
+    ::
+        An {{MLOperand}}. Specifies the N-D tensor of the scaling values whose shape is determined by the |axes| member in that each value in |axes| indicates the dimension of the input tensor with scaling values. For example, for an |axes| values of [1,2,3], the shape of this tensor is the list of the corresponding sizes of the input dimension 1, 2 and 3. When this member is not present, the scaling value is assumed to be 1.
+
+    : <dfn>bias</dfn>
+    ::
+        An {{MLOperand}}. Specifies the N-D tensor of the bias values whose shape is determined by the |axes| member in that each value in |axes| indicates the dimension of the input tensor with bias values. For example, for an |axes| values of [1,2,3], the shape of this tensor is the list of the corresponding sizes of the input dimension 1, 2 and 3. When this member is not present, the scaling value is assumed to be 0.
+
+    : <dfn>axes</dfn>
+    ::  
+        A sequence of {{unsigned long}}. The indices to the input dimensions to reduce. When this member is not present, it is assumed to be [1,2,3] that is, the reduction for the mean and variance values are calculated across all the input features for each individual sample in the batch.
+
+    : <dfn>epsilon</dfn>
+    ::
+        A {{float}} scalar. Specifies a small value to prevent computational error due to divide-by-zero.
+</dl>
+
+<div>
+    **Arguments:**
+        - *input*: an {{MLOperand}}. The input 4-D tensor.
```

Comment:
The input should be N-D tensor according to the algorithm step fix: https://github.com/webmachinelearning/webnn/pull/478/files/8d026d4f00c9c4d3952ce5f2939f35fda087cb17#r1410225434

Reply:
This should already be fixed.

Reply:
The latest file diff still shows "The input 4-D tensor" to me.

Reply:
```suggestion
        - *input*: an {{MLOperand}}. The input N-D tensor.
```

Reply:
Thanks. Fixed.